### DISCUSSION TITLE
Hard/Soft power-off function through DDC control?

### DISCUSSION THREAD

**Discussion by sarcomattys** [MonitorControl/MonitorControl#1035] [Ideas]
Hi, I am using MonitorControl for months, and it helped me out. May I suggest and expect some new function? :-) Thanks.

I am using three external monitors connecting to my mac mini in extended mode, but sometimes I did not want to use all of them together. Yes, sometimes I'd like to disable one or two monitors, without disconnecting the cords from the mac. So I think whether this function of hard/soft power off can be added in the future builds. 

I noticed an App called Luna, which can work on such things through DDC control. I am wondering if it might also work in MinitorControl.

URL: 
[https://www.reddit.com/r/MacOS/comments/n8274b/is_there_a_way_to_disable_an_external_monitor/](url)
[https://lunar.fyi/](url)

Thanks.

**Comment by waydabber** [comment]
Hi, Lunar can't disable a display - what it does is that it hides the display behind a mirror, turns the colors down to black so the screen is not visible and reduces the backlight as much as possible on the given display. There is no known way to really disconnect a display on macOS.

**Reply by waydabber** [reply]
(But some displays do support DDC power controls, this is not universal however.)

**Reply by sarcomattys** [reply]
I checked in Luna, and yes, it just dimmed and hid the display behind a mirror. Thanks for the reply.

**Reply by wasiq-me** [reply]
But Luna also ports the open application to the other screen if any.
For example, I have browser open on the main screen. Upon turning off the display, it will open the browser onto the secondary display and turn off the main one. I think we should do it as well.

**Reply by waydabber** [reply]
Yes, when a display becomes a mirror target, macOS automatically transports all windows that were on that display to an other display (as that display area is removed from the global display space when it becomes a mirror target). This is a generic macOS behavior, Lunar simply activates mirroring and dims/blacks out the screen.

**Comment by CodeConvergence** [comment]
Thank you for this useful and free app!

A quick question related to screens - using my MacBook (Catalina), can a kill (not hide etc.) the the local display and only use an external monitor? 

Thanks!
